src/runtime-rs/crates/hypervisor/src/qemu/qmp.rs

```diff
@@ -375,6 +375,103 @@ impl Qmp {
             }
         }
     }
+
+    #[allow(dead_code)]
+    pub fn hotplug_network_device(
+        &mut self,
+        netdev: &Netdev,
+        virtio_net_device: &DeviceVirtioNet,
+    ) -> Result<()> {
+        debug!(
+            sl!(),
+            "hotplug_network_device(): PCI before {}: {:#?}",
+            virtio_net_device.get_netdev_id(),
+            self.qmp.execute(&qapi_qmp::query_pci {})?
+        );
+        self.find_free_slot()?;
```

Comment:
I was wondering why this is called not having return values like `let (bus, slot) = self.find_free_slot()?;` below?

Reply:
This looks like a leftover.

Reply:
or maybe the invocation below should go here ?

Reply:
I'll remove call, thanks.  It is a leftover from early stages of development.